src/components/form/examples/form-layout.tsx

```diff
@@ -4,6 +4,49 @@ import { schema } from './schema-layout';
 
 /**
  * Layout
+ * By default, each item in a limel-form will be rendered in a single row; and
+ * each row occupies the entire available width of the form's container.
+ *
+ * This default layout may work fine on small screens or narrow containers,
+ * but on larger screens it won't usually produce a nice layout. Thus we
+ * recommend that you choose an appropriate responsive layout for your form.
+ *
+ * ###### Enabling responsive layouts
+ *
+ * By specifying a `grid` layout `type` in your `schema`, as well as your desired
+ * number of `columns`, you can leave the job of responsively handling the form
+ * layout to Lime Elements.
+ * ````ts
+ * export const schema = {
+ *     type: 'object',
+ *     lime: {
+ *         layout: {
+ *             type: 'grid',
+ *             columns: 3,
+ *         },
+ *     },
+ *     …
+ * };
+ * ````
+ * :::note
+ * Value for `columns` can only be `5`, `4`, `3`, `2`, or `1`. If you do not
+ * specify a value, `limel-form` will choose `5` by default.
+ * :::
+ *
+ * So if you have chosen `4` for instance, the form will do its best to fit in
+ * four columns in a row. But for smaller containers in which placement of four
+ * items per row is not possible, the form will automatically change the layout
+ * and fit in 3 items per row. As the container's width decreases, number of
+ * columns will also decrease.
+ *
+ * :::tip
+ * You can divide a form into collapsible or none-collapsible sections,
+ * and for each section specify a different layout.
+ * :::
+ *
+ * In this example, each collapsible section has its on `colSpan`.
+ * However, since the layout is responsive, make sure to change the browser
+ * window size to see how their responsive layout changes.
```

Comment:
No not at the moment, not sure how easy that will be to fix, it's probably mostly a CSS issue so @Kiarokh might have a better answer :slightly_smiling_face: 
As a workaround, while not perfect, you could always put the form in a full screen dialog

Reply:
Good point! Can we load the examples in **fullscreen dialogues** instead?

Reply:
Ah just saw that @jgroth have also written that 😂

Reply:
I'd say let's postpone that for now. The docs tell the reader to try different window sizes anyway.

Reply:
It's a great 💡! 

And it'd be nice to have an option in Kompendium to toggle an example to fullscreen and back its original size. (if that's what you also mean)

Reply:
Yeah I have thought about adding that for a while, a small button next to each example that will just display the example and nothing else. I think it would be great when you are developing or debugging something as there will be less noise on the screen and less stuff to load :slightly_smiling_face:

Reply:
it should be easy to do. Just give me a toggle-able class on the container. I'll do the rest. Also merge my last PR in Kompendium already 😄 !

Reply:
Haha yeah sorry, I will take a look at it. I've been less motivated to work on it now when I work from home all the time :disappointed:

Reply:
I hope neither of you feel like you can only work on Kompendium on your spare time. We use it in our work at Lime, so changes motivated by what we do at Lime can of course be developed and contributed as part of our work at Lime 🙂

I want to start working a bit on Kompendium too, but haven't felt like I've had the time for it yet. I guess I'll just have to make time at some point 😄

Reply:
Good point! It's something that Lime is directly benefiting from. Even if it wasn't, you could still argue that it brings free publicity for the company 😄 

But we should present it decently also, (which was my latest PR in Kompendium).